toolsHCK.ps1

```diff
@@ -901,6 +901,7 @@ function createprojecttarget {
                 $WntdtoTarget.Add($WntdTarget) | Out-Null
             }
             foreach ($toTarget in $WntdtoTarget) {
+                if ($WntdPITargets | Where-Object { $_.Key -eq $toTarget.Key }) { throw "The target is already being targeted in the project, aborting..." }

```

Comment:
Before this change, it was possible to target targets that are already being targeted, this creates duplicates of the same targets and it is not practical.